src/mixins/window-width-mixin.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  data: () => {
+    return {
+      winWidth: 0,
```

Comment:
Can you rename this to `windowWidth`? I'd prefer to describe variable names in full, unless it's an acronym.

Reply:
change made